launcher/ui/pages/instance/InstanceSettingsPage.cpp

```diff
@@ -75,6 +80,7 @@ bool InstanceSettingsPage::shouldDisplay() const
 InstanceSettingsPage::~InstanceSettingsPage()
 {
     delete ui;
+    delete accountMenu;
```

Comment:
```suggestion
```
since you've created the QMenu with a parent, it will be deleted with the window anyway, you don't have to do it manually.